presto-docs/src/main/sphinx/sql/create-function.rst

```diff
@@ -0,0 +1,83 @@
+===============
+CREATE FUNCTION
+===============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    CREATE [ OR REPLACE ] FUNCTION
+    qualified_function_name (
+      parameter_name parameter_type
+      [, ...]
+    )
+    RETURNS return_type
+    [ COMMENT function_description ]
+    [ LANGUAGE language ]
+    [ DETERMINISTIC | NOT DETERMINISTIC ]
+    [ RETURNS NULL ON NULL INPUT | CALLED ON NULL INPUT ]
+    [ RETURN expression ]
+
+
+Description
+-----------
+
+Create a new function with the specified definition.
+
+Each function is uniquely identified by its qualified function name
+and its parameter type list. ``qualified_function_name`` needs to be in
+the format of ``catalog.schema.function_name``.
+
+To create a function, the function namespace as identified by
+``catalog.schema`` needs to managed by a
+:doc:`/admin/function-namespace-managers`.
+
+The optional ``OR REPLACE`` clause causes the ``ALREADY_EXISTS`` error
+to be suppressed if the function already exists.
+
+The ``return_type`` needs to match the actual type of the routine body
+``expression``, without performing type coercion.
+
+A set of routine characteristics can be specified to decorate the
+function and modify its behavior. Each kind of routine
+characteristic can be specified at most once.
+
+============================ ======================== ================================================================
+Routine Characteristic       Default Value            Description
+============================ ======================== ================================================================
+Language clause              SQL                      The language in which the function is defined.
+Deterministic characteristic NOT DETERMINISTIC        Whether the function is deterministic. ``NOT DETERMINISTIC``
+                                                      means that the function is possibly non-deterministic.
+Null-call clause             CALLED ON NULL INPUT     The behavior of the function in which ``null`` is supplied as
+                                                      the value of at least one argument.
```

Comment:
Similarly here - please specify default

Reply:
The second column shows the default value.